GettingStarted_4.0.0-rc2.md

```diff
@@ -48,6 +49,20 @@ Change 5:
       - New argument “struct xNetworkInterface * pxNetworkInterface” added.
       - ipconfigIPv4_BACKWARD_COMPATIBLE flag is used to differentiate between old API and new API.
 
+Change 6:
+
+   - Existing API: xApplicationDHCPHook
+   - New API: xApplicationDHCPHook_Multi
+      - New argument "struct xNetworkEndPoint * pxEndPoint" added.
+      - ipconfigIPv4_BACKWARD_COMPATIBLE flag is used to differentiate between old API and new API.
+
+Change 7:
+
+   - Existing API: xApplicationDNSQueryHook
+   - New API: xApplicationDNSQueryHook_Multi
+      - New argument "struct xNetworkEndPoint * pxEndPoint" added.
+      - ipconfigIPv4_BACKWARD_COMPATIBLE flag is used to differentiate between old API and new API.
```

Comment:
Can we document about the usage `ipconfigIPv4_BACKWARD_COMPATIBLE` flag in the beginning/end of the changes list instead of specifying it repeatedly only for change 5,6, and 7 as its applicable for all the changes.

Reply:
Done